pro.saladin.Eeman.json

```diff
@@ -0,0 +1,99 @@
+{
+  "id": "pro.saladin.Eeman",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "48",
+  "provides": { "id": "pro.saladin.Eeman" },
+  "replaces": { "id": "pro.saladin.Eeman" },
```

Comment:
not sure what documentation say these two lines are valid, but that documentation is wrong.

Reply:
Oh wait there is a typo.. i will fix this too once i get back home

Reply:
the bigger issue is that it needs to be in a wholly different file.

Reply:
Added it https://github.com/SaladinAyyub/Eeman/blob/main/data/pro.saladin.Eeman.metainfo.xml.in